Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Formats/Format.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats
+{
+    public enum Format
```

Comment:
That would mean that the user would have to add their own code to achieve the same functionality, which would defeat the point of having a control that is universal as per discussion at #1107. If not using a formatter class, how do you propose supporting both Rich Text and Markdown? With what you are saying, the user might as well just use a CommandBar and create their own from scratch.